lib/traject/yale_csv_config.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+# Yale CSV Mapping Configuration
+require_relative 'csv_reader'
+require_relative 'dlme_json_resource_writer'
+require_relative 'macros/dlme'
+require_relative 'macros/csv'
+extend Macros::DLME
+extend Macros::Csv
+
+settings do
+  provide 'writer_class_name', 'DlmeJsonResourceWriter'
+  provide 'reader_class_name', 'CsvReader'
+end
+
+to_field 'agg_provider',     provider
+to_field 'id',               normalize_prefixed_id('OrbisBibID')
+to_field 'cho_identifier',   column('OrbisBibID', trim: true)
+to_field 'cho_provenance',   column('Place of origin', trim: true)
+to_field 'cho_creator',      column('Creator', split: '|', trim: true)
+to_field 'cho_title',        column('Title', split: '|', trim: true)
+to_field 'cho_alternative',  column('Variant Titles', split: '|', trim: true)
+to_field 'cho_publisher',    column('Publisher', trim: true)
+to_field 'edm:dateProvider', column('Date, created', trim: true)
```

Comment:
I don't think this is the right IR field.

Reply:
This should probably be `cho_date`?

Reply:
it should be. was commented on in the spreadsheet mapping as well. i think wayne is trying to differentiate between key dates + provided / raw / text dates, but we don't have support for that in the IR MAP right now.